installer/Migration/V5_2_0/permission/data_group.yaml

```diff
@@ -0,0 +1,41 @@
+dashboard_admin_widgets:
```

Comment:
better to split `subunit`, `location`
`dashboard_subunit_widgets`
`dashboard_location_widgets`